stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/factory/StreamOfflinePluginFactory.kt

```diff
@@ -158,6 +162,16 @@ public class StreamOfflinePluginFactory(
         ).also { offlinePlugin -> cachedOfflinePluginInstance = offlinePlugin }
     }
 
+    private fun getEditMessageListener(chatClient: ChatClient, statePlugin: StatePlugin): EditMessageListenerComposite {
+        val editMessageListenerDatabase = EditMessageListenerDatabase(
+            userRepository = chatClient.repositoryFacade,
+            messageRepository = chatClient.repositoryFacade,
+            clientState = chatClient.clientState
+        )
```

Comment:
Would you mind passing the exact parameters to avoid the amount of `chatClient.repositoryFacade` usage? It makes manual code analysis a bit harder.
```suggestion
    private fun getEditMessageListener(
        clientState: ClientState, 
        repositoryFacade: RepositoryFacade, 
        statePlugin: StatePlugin
    ): EditMessageListenerComposite {
        val editMessageListenerDatabase = EditMessageListenerDatabase(
            userRepository = repositoryFacade,
            messageRepository = repositoryFacade,
            clientState = clientState
        )
```